src/app/_components/BackButton/index.tsx

```diff
@@ -1,10 +1,16 @@
 // TODO: update href to filter for the same category
 import Link from 'next/link'
+import BackArrow from "../../_icons/BackArrow";
 
-export default function BackButton() {
+import styles from './styles.module.css'
+
+export default function BackButton({className}) {
   return (
-    <>
-      <Link href="/">Back</Link>
-    </>
+    <div className={className}>
```

Comment:
This div is redundant right? The className could be on the Link

Reply:
changing it breaks the visuals. this means i should be encapsulating `BackButton` on a div instead of having it be div?

examples below:

with the <div>

![image](https://github.com/user-attachments/assets/eeeef3ab-46de-4aa2-88be-ac544a5aae11)

without the <div>

![image](https://github.com/user-attachments/assets/58cc4e6c-1aba-4103-a455-3cf06e8c1a83)

would creating a `<div className={styles.backButtonContainer}>` on the `page.tsx` file make more sense?

Reply:
Ahh I see, that's because <a> has display: inline by default, if you change it to block it should work the same (I actually though the CSS reset did this)

Reply:
oh i see! 
the CSS reset is not working as intended on this one as I've added it on the `:root` and it's messing class attribution. 
Upcoming PR #37 address these issues.